iothub/device/src/DeviceClient.cs

```diff
@@ -35,143 +33,143 @@ private DeviceClient(InternalClient internalClient)
         }
 
         /// <summary>
-        /// Create an Amqp DeviceClient from individual parameters
+        /// Creates a disposable, Amqp DeviceClient from the specified parameters
```

Comment:
Does this have to AMQP? This is legacy documentation perhaps

Reply:
Actually no, disregard this. AMQP is the default protocol selected I think

Reply:
It defaults to Amqp, yes